src/Ecal/EcalVetoProcessor.cxx

```diff
@@ -89,6 +104,13 @@ void EcalVetoProcessor::buildBDTFeatureVector(
   bdtFeatures_.push_back(result.getAvgLayerHit());
   bdtFeatures_.push_back(result.getDeepestLayerHit());
   bdtFeatures_.push_back(result.getStdLayerHit());
+  // MIP tracking (unused but could be uncommented)
```

Comment:
I think it might be best to double check with Anthony about this, actually, since I know that he's done some studies on including the tracking variables in the BDT.  I'll send him a message and ask what he's planning on including the BDT.

Reply:
Seems like Anthony's definitely planning on using them in the seg-mip BDT changes, but he's going to do a more thorough study before officially merging everything in.  It's up to you whether you want me to make the use of tracking variables in the BDT a configuration variable, although it might be easier to wait until after the study.

Reply:
No worries, we can wait until the BDT is "officially" updated.